catalog/styles/header.css

```diff
@@ -0,0 +1,332 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-top: 54px;
+}
+
+.header__menu {
+  display: flex;
+  justify-content: space-between;
+  max-width: 477px;
+  width: 100%;
+}
+
+.header__logo-link:hover {
+  opacity: 0.6;
+}
+
+.header__logo-link:active {
+  opacity: 0.3;
+}
+
+.header__link {
+  font-size: 20px;
+  color: #111111;
+}
+
+.header__link:hover {
+  opacity: 0.6;
+}
+
+.header__link:active {
+  opacity: 0.3;
+}
+
+.header__link--active {
+  position: relative;
+}
+
+.header__link--active::after {
+  content: "";
+  position: absolute;
+  top: 35px;
+  left: 0;
+  width: 100%;
+  height: 2px;
+  background-color: #68b738;
+}
+
+.main-screen__content {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  position: relative;
+  left: 83px;
+  bottom: 24px;
+}
+
+.main-screen__product-pic {
+  margin-top: 84px;
+}
+
+#header__nav--list {
+  display: flex;
+  justify-content: space-between;
+  width: 477px;
+  max-width: 100%;
+}
+
+#header__nav__toggle {
+  visibility: hidden;
+}
+
+#header__nav--list {
+  visibility: visible;
+}
+
+.main-screen__product-desktop {
+  margin: 22px 0 60px 0;
+}
+
+.main-screen__product-tablet {
+  position: absolute;
```

Comment:
When you set position to absolute, always specify horizontal and vertical parameters, at least one per axis — `top` or `bottom`, `left` or `right`.

UPD. Although... It all looks like a `visually-hidden` class description. If so, you already have a `visually-hidden` class somewhere in your CSS — simply add it to this element in HTML, instead of duplicating this code.